src/main/resources/org/akaza/openclinica/i18n/page_messages.properties

```diff
@@ -725,7 +725,7 @@ subject_event_cannot_signed = The Participant Event is not ready to be signed.
 
 subject_event_data_is_as_it_was_before = Participant Event data is as it was before it was locked - available for viewing, querying, and extraction, as well as for additions and modifications.
 
-subject_event_signed = All the Events in the Participant have been updated to a status of "signed".
+subject_event_signed = All the Events in the Participant <Participant ID> have been updated to a status of "signed".
```

Comment:
use the same pattern as "subject_reassigned" below and use MessageFormat to replace with value.

Reply:
@svadla-oc OK changed to using MessageFormat on latest commit